cmd/ipfs-cluster-service/main.go

```diff
@@ -279,7 +279,7 @@ configuration.
 					Name:   "monitor",
 					Value:  defaultMonitor,
 					Hidden: true,
-					Usage:  "peer monitor to use [basic,pubsub].",
+					Usage:  "peer monitor to use [pubsub].",
```

Comment:
remove this option altogether

Reply:
Seems like this was not being used. I didn't find `c.String("monitor")`

Reply:
because you removed `setupMonitor` in this PR